change/@fluentui-react-charting-20c8ae70-f2a4-4ef4-89b6-7ca932f7481d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28414&commitId=270dafaad594e0e8ae44c174f527d27573852465&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- contain, image taller.default.chromium.png|1186| Changed|
|Image.Fit- cover, image taller.default.chromium.png|7603| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>